theseus/theseus_layer.py

```diff
@@ -264,6 +261,7 @@ def _instantiate_dlm_bwd_objective(objective):
 
     bwd_optimizer = GaussNewton(
         bwd_objective,
+        vectorize=False,
```

Comment:
We should have vectorize on for DLM as well?

Reply:
I changed it to `True` and it passes the backward tests, I'll be more comfortable with this part after we re-run the backward experiments, but I'll leave as is for now.